src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -220,16 +222,29 @@ private async Task<string> Fetch(Func<HttpRequestMessage> requestFactory, string
                 return value404;
             }
 
-            return await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync();
+            await EnsureSuccessStatusCode(response);
```

Comment:
keep the existing way of throwing the exception:
```suggestion
      try
      {
            return await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync();
      }
      catch (Exception ex)
      {
          ex.Data["RequestUrl"] = requestUrl;
          throw;
      }
```